### ISSUE TITLE
Card with States - Export to pdf issue

### ISSUE THREAD

**Issue by rsathish26** [okviz/free-visuals#66]
Hi, We are using the Card with States version 1.4.2.0 downloaded from your website. The visual is added our organisational view and then being used in PBI reports. The issue is the visual does not get exported into pdf after the report been published to the workspace. It export works well from the desktop but not after it has been published. Can you please let us know if this a bug and if so when is it likely to be fixed
Thank you

**Comment by danieleperilli** [comment]
You have download a beta version from our website that is not certified, so it is not rendered on PDF/Power Point. Just download the certified version from the marketplace: https://store.office.com/addinsinstallpage.aspx?assetid=WA104380967

**Comment by rsathish26** [comment]
Hi Daniele,

We have checked and this is not the beta version. The link below was the same one from where the visual was downloaded and added to the organisation visual.
I have also created a new PBI report and added the visual and have checked the version.
As mentioned, its not an issue exporting to pdf from PBI desktop. The issue is only when it’s published on to the workspace and then exported to pdf.
Appreciate your help!

Thank you,
Rajshri
